index.d.ts

```diff
@@ -0,0 +1,44 @@
+export type Reviver = (key: string, value: any) => any;
+export type BeforeParse = (data: string) => string;
+
+export interface Options {
+	/**
+	 * Applies a function to the JSON string before parsing.
+	 */
+	beforeParse?: BeforeParse;
+	/**
+	 * Prescribes how the value originally produced by parsing is transformed, before being returned.
+	 * See the [JSON.parse docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#Using_the_reviver_parameter) for more.
+	 */
+	reviver?: Reviver;
+}
+
+/**
+ * Read and parse a JSON file.
+ *
+ * Strips UTF-8 BOM, uses graceful-fs, and throws more helpful JSON errors.
+ *
+ * @example
+ * import * as loadJsonFile from 'load-json-file';
+ *
+ * const json = loadJsonFile.sync('foo.json')
+ * console.log(json);
+ * //=> {foo: true}
+ */
+export function sync<T = any>(filepath: string, options?: Options): T;
+
+/**
+ * Read and parse a JSON file.
+ *
+ * Strips UTF-8 BOM, uses graceful-fs, and throws more helpful JSON errors.
+ *
+ * @example
+ * import loadJsonFile from 'load-json-file';
+ *
+ * (async () => {
+ * 	const json = await loadJsonFile('foo.json');
+ * 	console.log(json);
+ * 	//=> {foo: true}
+ * })();
+ */
+export default function loadJsonFile<T = any>(filepath: string, options?: Options): Promise<T>;
```

Comment:
Should we use `unknown` here?

Reply:
🤔 `unknown` isn't a type in Typescript, `any` would be considered the "unknown" since this type can be applied to anything.

Reply:
Oh I didn't know this was a new type in Typescript 3.x

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html

Learn something every day!